### ISSUE TITLE
Conditional agree in the First 10 sec

### ISSUE THREAD

**Issue by unzer** [Fruitfella/ROBOT-binary#49]

**Comment by unzer** [comment]
Is It possible run a conditional in the First 10 sec of an open candle? After this period, wait for the next candle open...?

**Comment by Fruitfella** [comment]
very possible 😄

**Comment by unzer** [comment]
Hi @Fruitfella 
Tks for the answer.
Probabily I need to calculate [candle 1 entiry time]-[candle 1 open or open time], Is it correct my logic?

**Comment by Fruitfella** [comment]
you know how to purchase at candle open (new candle) right?
what you need to do is : 
set a variable with **true** value when candle open, then set it **false** 10s later.
![capture](https://user-images.githubusercontent.com/26277327/28899106-7d6ff4cc-7813-11e7-9ef1-1633e1585471.png)

**Comment by unzer** [comment]
Tks a lot, i Will test and come back...

Em 2 de ago de 2017 8:18 PM, "Fruitfella" <notifications@github.com>
escreveu:

**Comment by unzer** [comment]
Hi @Fruitfella 
Take a look in the adapt that I did in your code.
So, when the last candle closed, I start to count until 10 seconds, If during this period the RSI are less then 100, purchase rise... but this purchase can only happing during the first 10 seconds...
But this is not happing, they are do a looping.... what I did wrong?

![image](https://user-images.githubusercontent.com/26580460/28997019-caa90d3a-79e1-11e7-9c24-dfdc71aa639d.png)

**Comment by Fruitfella** [comment]
That initial candle is wrong, dont you have **`Is new candle"** or any other **new candle tool** ?
That trade time also wrong.

**Comment by unzer** [comment]
@fruitfella
I dont have this option (is New candle?)
I need to update my bot?


Em 5 de ago de 2017 1:43 PM, "Fruitfella" <notifications@github.com>
escreveu:

That initial candle is wrong, dont you have *`Is new candle"* or any other *new
candle tool* ?
That trade time also wrong.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Fruitfella/ROBOT-binary/issues/49#issuecomment-320453915>,
or mute the thread

**Comment by Fruitfella** [comment]
nope, you can get it from <a href="https://gist.github.com/aminmarashi">amin's gist</a> or mine here #35

**Comment by Fruitfella** [comment]
oh.. im dead tired, gonna take a rest. cya unzer
good luck with your bot
